max_lin_speed: 5.0
min_lin_speed: -5.0
lin_step: 0.5

max_ang_speed: 6.0
ang_steps: 60
step_len: 1.0 #seconds

dead_man_index: 5
